### DISCUSSION TITLE
Deformable Sprite custom deformation doesn't mirror on flipped limb

### DISCUSSION THREAD

**Discussion by TheNinjaScout** [FakeFishGames/Barotrauma#15434] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

Possible fix in PR https://github.com/FakeFishGames/Barotrauma/pull/15456

Adding horizontal custom deformation to a limb results results in the deformation being applied as if the limb wasn't flipped when the character is flipped

https://github.com/user-attachments/assets/264702a0-7bbd-4c8f-8137-50746467c791

(In the example I changed the crawler's Torso limb's custom deformation to include some horizontal deformation, which visibly behaves differently when the character is flipped, this also happens if the custom deformation is used by itself rather than multiplying an inflate deformation as it currently is doing)
```xml
<SpriteDeformation type="custom" resolution="5,5" frequency="0" blendmode="Multiply" row0="45,0 45.5,1 1,1 0.5,1 0,0" row1="0,0 0.5,1 1,1 0.5,1 0,0" row2="0,0 0.5,1 1,1 0.5,1 0,0" row3="0,0 0.5,1 1,1 0.5,1 0,0" row4="0,0 0.5,1 1,1 0.5,1 0,0" amplitude="1" sync="-1" typename="custom" usemovementsine="False" stopwhenhostisdead="False" sineoffset="0" strength="1" maxrotation="90" onlyinwater="False" />
```

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This has been addressed in our private repo now, fix coming up in the next unstable build and the upcoming release.
